### DISCUSSION TITLE
Full screen

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1885] [Help]
Hello everyone, 

Is it possible in Go, when you have a touch screen, to have the stops panels full screen and not just "enlarged" on the screen?

that is to say see just the panels without the upper window bar (the one with the minimize, maximize and close window buttons), as well as the task bar at the bottom of the screen?

Thanks !

**Comment by oleg68** [comment]
Now it is not possible. Pannels always have the title bar.

**Reply by Jean-Gabriel1982** [reply]
Ah .. is it possible to develop this feature then ? 
it'd be great ! 🙂
